.travis.yml

```diff
@@ -3,8 +3,8 @@ cache: cargo
 rust:
   - stable
   - beta
-  - nightly
-  - 1.36.0
+#  - nightly
```

Comment:
Why is testing on nightly no longer being done? This looks WIP

Reply:
Because nightly panics (which is a compiler bug). This is to be expected as it's nightly.

Reply:
I'm not expecting that. I have been building my crates on nightly in CI for years and very rarely have issues. But OK. I can't find the travis log right now.